builds/msvc/vs2013/libbitcoin-examples/libbitcoin-examples.vcxproj.filters

```diff
@@ -8,17 +8,11 @@
 <Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <ItemGroup>
     <Filter Include="resource">
-      <UniqueIdentifier>{B726DF7D-6D1D-48FB-0000-000000000003}</UniqueIdentifier>
+      <UniqueIdentifier>{B726DF7D-6D1D-48FB-0000-000000000001}</UniqueIdentifier>
     </Filter>
     <Filter Include="src">
       <UniqueIdentifier>{B726DF7D-6D1D-48FB-0000-000000000000}</UniqueIdentifier>
     </Filter>
-    <Filter Include="src\.deps">
-      <UniqueIdentifier>{B726DF7D-6D1D-48FB-0000-000000000001}</UniqueIdentifier>
-    </Filter>
-    <Filter Include="src\.libs">
-      <UniqueIdentifier>{B726DF7D-6D1D-48FB-0000-000000000002}</UniqueIdentifier>
-    </Filter>
```

Comment:
@pmienk Could you update the build system to prevent this?

Reply:
Will look at it now.  Looks like you'd like to exclude anything 'hidden' - from both file and directory listings?

Reply:
There are a few hidden files that we use, but no hidden directories. Not sure if the hidden files are relevant to the source listings though.

Reply:
I had some errand .gitignore show up in one file listing, so I'm hoping we can ignore all of them or I can track it down and remove it.  Its contents can move into the top-level one.  But I'll rewrite the discovery function to allow for directory and file ignoring as separate parameters.

Reply:
Addressed by libbitcoin/libbitcoin-build#164.  I did not see any use of hidden files and so have currently excluded them.  Also finally pulled the file discovery code into utilities (it's also used in my in-progress cmake build)